deployments/k8s/helm/signalfx-agent/templates/secrets.yaml

```diff
@@ -10,4 +10,4 @@ metadata:
     release: {{ .Release.Name }}
 type: Opaque
 data:
-  access-token: {{ .Values.signalFxAccessToken | b64enc | quote }}
+  access-token: {{ (required ".Values.signalFxAccessToken is required. If you need an access token visit https://www.signalfx.com for a free trial." .Values.signalFxAccessToken) | b64enc | quote }}
```

Comment:
If you need an access token, check your profile page in the SignalFx web application, or visit....